pipeline/clean/opuscleaner/configs/en-zh/WikiMatrix-v1.filters.json

```diff
@@ -3,6 +3,126 @@
   "files": [
   ],
   "filters": [
+    {
```

Comment:
Are all these config changes coming from your CJK training run?

Reply:
Jaume added these filters based on his manual experiments in OpusCleaner, and we used them for training. What we missed is that the rest of the default filters should also be present. I'm fixing it here by copying them from default.config.json for CJK.